README.md

```diff
@@ -172,6 +173,30 @@ Funtoo Linux has an autogenerated lazygit package in [dev-kit](https://github.co
 ```sh
 sudo emerge dev-vcs/lazygit
 ```
+### Gentoo Linux
+
+Lazygit is not (yet) in main Gentoo portage, however an ebuild is available in [cova overlay](https://github.com/cova-fe/cova-overlay/tree/main/dev-vcs/lazygit)
+
+You can either add the overlay to your system and install lazygit as usual:
+
+```sh
+sudo eselect repository enable cova
+sudo emaint sync -r cova
+sudo emerge dev-vcs/lazygit
+```
+
+Or you can download the ebuild and install it manually; please consider the example below just as a suggestion to be adapted to your system.
+
+```sh
+su
+LAZYGIT_VERSION="0.39.2" # Replace with the version you want from cova-overlay
```

Comment:
```suggestion
LAZYGIT_VERSION="0.39.3" # Replace with the version you want from cova-overlay
```
May as well set to the latest version as of 6 hours ago :)

Reply:
Fair enough, I update the ebuild but forgot the PR :)